src/main/java/redis/clients/jedis/Connection.java

```diff
@@ -37,6 +37,7 @@
   private SSLSocketFactory sslSocketFactory;
   private SSLParameters sslParameters;
   private HostnameVerifier hostnameVerifier;
+  private JedisCommandListener listener = NoopJedisCommandListener.INSTANCE;
```

Comment:
Isn't it better to have EventListenerList?

Reply:
I believe this can result in an unnecessary dependency on `java.desctop` jigsaw module which can result in issues for library users on JDK 9+ and use compact java images.

Reply:
Perhaps it doesn't need to be EventListenerList but a the idea of managing a list of listeners I think is more robust

Reply:
That makes sense.  I'll update to supporting a `List<JedisCommandListener>`, with the default being an empty list.

Reply:
OK, I've pushed a commit that supports multiple listeners @gkorland .  Please let me know what you think.

Reply:
Hey, @gr8routdoors! Wdyt of using `ArrayList` instead? In the vast majority of use cases, it allows for better performance and memory usage. There is a good discussion regarding this topic here: https://stackoverflow.com/questions/322715/when-to-use-linkedlist-over-arraylist-in-java

Reply:
Hi! My assumption was that the number of listeners would be very small, so I chose `LinkedList` for a smaller memory footprint.  Could easily change it to `ArrayList` if you think it better matches the expected usage.